src/components/Layout/components/makeDataUiTheme.ts

```diff
@@ -0,0 +1,14 @@
+import merge from 'lodash/merge';
+import {DATA_UI_LAYOUT_THEME} from '../constants';
```

Comment:
"data-ui "is not a correct naming for external projects, it's more like "default" theme